apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -413,11 +415,76 @@ post_correct_tx(_Config) ->
             fee => Fee,
             nonce => Nonce}),
     {ok, SignedTx} = rpc(aec_keys, sign, [SpendTx]),
-    post_tx(SignedTx),
+    {ok, 200, _} = post_tx(base64:encode(aec_tx_sign:serialize_to_binary(SignedTx))),
     {ok, [SignedTx]} = rpc(aec_tx_pool, peek, [infinity]), % same tx 
     ok.
 
-%% TODO: POST broken tx (currently not handled in API Endpoint)
+post_broken_tx(_Config) ->
+    Amount = 7,
+    Fee = 2,
+    BlocksToMine = 10,
+    aecore_suite_utils:mine_blocks(aecore_suite_utils:node_name(?NODE),
+                                   BlocksToMine),
+    {ok, []} = rpc(aec_tx_pool, peek, [infinity]), % empty 
+    {ok, 200, _} = get_balance(), % account present
+    {ok, PubKey} = rpc(aec_keys, pubkey, []),
+    {ok, Nonce} = rpc(aec_next_nonce, pick_for_account, [PubKey]),
+    {ok, SpendTx} =
+        aec_spend_tx:new(
+          #{sender => PubKey,
+            recipient => random_hash(),
+            amount => Amount,
+            fee => Fee,
+            nonce => Nonce}),
+    {ok, SignedTx} = rpc(aec_keys, sign, [SpendTx]),
+    [T, V, SerializedTx, Sigs] = aec_tx_sign:serialize(SignedTx),
+    lists:foreach(
+        fun(Key) ->
+            BrokenTx = lists:filter(
+                fun(#{Key := _}) -> true;
+                (_) -> false
+                end,
+                SerializedTx),
+            EncodedBrokenTx = base64:encode(msgpack:pack([T, V, BrokenTx, Sigs])),
```

Comment:
I see this test tests that attempting to post a (base64-encoded) signed tx with the tx stripped of all except one of its fields is detected as broken. There are many ways a signed tx can be broken:
* Removed or added or shuffled fields in the signed tx envelope;
* Amended field values in the signed tx envelope;
* Removed or added or shuffled fields in the spend tx;
* Amended field values in the spend tx.

This approach for making a tx broken is one of the many possible, and is good enough. So no changes needed on this.

----

I also see this test has the implied assumption that the hardcoded list of fields below (type, vsn, sender, recipient, amount, fee, nonce) is the full list of fields of the spend tx, that is correct though makes the test fragile to future tx internal format changes. I recommend hardcoding this expectation in the test so to ease future updates of the test in case of future tx internal format change.